manuscript.Rmd

```diff
@@ -854,14 +857,41 @@ Fourth, despite its rigor, PAC does accommodate data-dependent decisions if thes
 Researchers can, for example, formulate conditions (e.g., in the form of if-else-blocks) under which they prefer one analysis type over the other.
 For example, if distributional assumptions are not met, the code may branch out to employ robust methods; or, an analysis may perform automated variable selection mechanisms before running the final model.
 Another example of data dependent decisions are more explorative analyses, i.e. explorative factor analysis or machine learning.
+Decisions that do not lend themselves to formulation in code, e.g. visual inspection, must still be described verbally or be treated as noted in the section [Deviating from the preregistration and exploration].
 With PAC researchers can precisely calibrate how much they want to explore.
 When communicating a finding as explorative, it may still make a difference if tens or hundreds of variables were considered.
-Fifth, deviations from the preregistration are clearly documented, because they are reflected in changes to the code which are managed and tracked with version control.
-One could even make a commit labeled "Necessary deviations from the preregistration after data collection",
-and reference it in the manuscript.
-
-Note that preregistration, or PAC, does not preclude exploratory analyses.
-Such analyses should merely be explicitly declared as such in the manuscript.
+Fifth, deviations from the preregistration are clearly documented because they are reflected in changes to the code, which are managed and tracked with version control.
+
+## Deviating from the preregistration and exploration
+
+We would like to note that in the PAC paradigm, the direct comparison of the preregistration and the final publication is particularly easy to accomplish.
+Because there is a single unambiguous reference point, that is the dynamic document, including code, under version control, changes can be inspected by authors and readers alike.
+To make the comparison even more accessible, we highly recommend that the authors review all changes made to the preregistered plan to summarize and justify them for their reader, e.g. in the discussion of the final manuscript [^summarize-changes].
```

Comment:
we recommend that authors maintain a change log ...

Reply:
Your changelog idea is fantastic. I hesitated to make it an absolute requirement because it is an additional step but recommended it.
Instead of marking the commit message, I recommended logging the commit id in the changelog.